sbi/inference/abc/abc_base.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+from abc import ABC
+
+import torch
+from typing import Callable, Union
+
+from sbi.simulators.simutils import simulate_in_batches
+from sbi.user_input.user_input_checks import process_x, prepare_for_sbi
+from numpy import ndarray
+from torch import Tensor
+
+
+class ABCBASE(ABC):
+    def __init__(
+        self,
+        simulator: Callable,
+        prior,
+        x_o: Union[Tensor, ndarray],
```

Comment:
`x_o` should be passed at `__call__()` to be consistent with the other methods.

Reply:
This is a bit annoying because there is no amortization in ABC and we need `x_o` as attribute in any case But I will change it match the other methods.